requirements.txt

```diff
@@ -0,0 +1,17 @@
+# s3/boto, let aiobotocore decide botocore/boto3/awscli versions
+aiobotocore[boto3,awscli]
+python-dateutil==2.8.0
+matplotlib
+click
+scikit-image
+ruamel.yaml
+awscli
+jupyter
+folium
+# git+git://github.com/sat-utils/sat-search@0.3.0-b2
+git+git://github.com/sat-utils/sat-search
+
+--extra-index-url="https://packages.dea.ga.gov.au"
+datacube[performance,s3]
```

Comment:
This no longer pulls is `GDAL` need to add it manually for your indexing script

Reply:
Actually it would pull in GDAL, because this does not install latest datacube.
To install latest we now need `--pre` option.